### PR TITLE
Vet clinic database Project

### PR DESCRIPTION
**Pull request by Johnadibe**
#### Wrote queries for the following quests:
- Find all animals whose name ends in "mon" [Result Screenshot](https://user-images.githubusercontent.com/40009354/229091554-a04940b7-d3b3-455e-a415-736bee30e14b.PNG)
- List the name of all animals born between 2016 and 2019 [Result Screenshot](https://user-images.githubusercontent.com/40009354/229091853-d5dd068f-159d-4d35-a808-6817476abf32.PNG)
- List the name of all animals that are neutered and have less than 3 escape attempts [Result Screenshot](https://user-images.githubusercontent.com/40009354/229092425-436ac487-4e6c-42f6-a609-883e43d29055.PNG)
- List the date of birth of all animals named either "Agumon" or "Pikachu" [Result Screenshot](https://user-images.githubusercontent.com/40009354/229092641-394c3385-29b3-4f01-8913-8e29f223fdba.PNG)
- List name and escape attempts of animals that weigh more than 10.5kg [Result Screenshot](https://user-images.githubusercontent.com/40009354/229093140-1edb85b5-fd69-46e0-9f8b-67c33f2e9eb0.PNG)
- Find all animals that are neutered [Result Screenshot](https://user-images.githubusercontent.com/40009354/229093318-c632a96f-0caa-40ce-bea0-538869eecf36.PNG)
- Find all animals not named Gabumon [Result Screenshot](https://user-images.githubusercontent.com/40009354/229093547-8a0977b3-e9a6-4f61-8e79-9754493b3517.PNG)
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) [Result Screenshot](https://user-images.githubusercontent.com/40009354/229093724-02397c2a-bb43-44b9-8f38-f438939ab083.PNG)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- PROJECT 1
+SELECT * FROM animals WHERE name ILIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN DATE '2016-01-01' AND '2019-01-01';
```
Comment (AnselemOdims) [inline queries.sql]: - [ ] Good job working on this, but the date_of_birth should be between 2016 and 2019
Hint:
`2016-01-01 AND 2019-12-31`

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Johnadibe , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (Johnadibe) [reply queries.sql]: Okay Thanks @AnselemOdims.😊. I have made the changes.

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Johnadibe 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**